src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -48,7 +48,7 @@ public static class CLIOptions
 
         public static readonly CLIOption<string> ProjectName = new CLIOption<string>
         {
-            ArgumentName = "--project",
+            ArgumentName = "--project-file",
```

Comment:
Maybe it is confusing that the JsonKey is `project-name` and the argumentname is `--project-file`. I think they should be in line.

Reply:
An example of a similar change can be found in #132. You could use that as a reference.